packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -415,13 +431,6 @@ const main = (opts) => {
     }
 
     const OUTPUT_DIR_FLAG_ACTIVE = !(opts.outputDir === DEFAULT_OUTPUT_DIR)
```

Comment:
Is this constant still being used?

Reply:
yep, used here: https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-create-app/scripts/create-mobify-app.js#L450